### PR TITLE
Add hsy

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -281,6 +281,53 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		},
+		condition: {
+			noCopy: true,
+			duration: 2,
+			onStart(pokemon) {
+				this.add('-start', pokemon, 'Wonder Wing');
+			},
+			onRestart(target, source, sourceEffect) {
+				target.removeVolatile('wonderwing');
+			},
```
Comment (HisuianZoroark) [inline data/mods/ssb/moves.ts:313]: is this needed?

File: data/mods/ssb/moves.ts [hunk 2]
```diff
@@ -281,6 +281,53 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+			onDamage(damage, target, source, effect) {
+				if (this.effectState.duration < 2) return;
```
Comment (HisuianZoroark) [inline data/mods/ssb/moves.ts:315]: make sure this increments

Comment (KrisXV) [reply data/mods/ssb/moves.ts:315]: This decrements in the scripts by default

File: data/mods/ssb/moves.ts (hunk 1 above)
Comment (KrisXV) [reply data/mods/ssb/moves.ts:313]: Yes

**Pull request merged**